src/lib.rs

```diff
@@ -867,6 +867,40 @@
 //! }
 //! ```
 //!
+//! ### Flattening enums of subcommands
+//!
+//! It is also possible to combine multiple enums of subcommands into one.
+//! All the subcommands will be on the same level.
+//!
+//! ```
+//! # use structopt::StructOpt;
+//! # fn main() {}
+//! #[derive(StructOpt)]
+//! enum BaseCli {
+//!     Command1(Command1),
+//! }
+//!
+//! #[derive(StructOpt)]
+//! struct Command1 {
+//!     arg1: i32,
+//! }
+//!
+//! #[derive(StructOpt)]
+//! struct Command2 {
+//!     arg2: i32,
+//! }
+//!
+//! #[derive(StructOpt)]
+//! enum Opt {
+//!     #[structopt(flatten)]
+//!     BaseCli(BaseCli),
+//!     Command2(Command2),
+//! }
+//! ```
+//!
+//! + `cli command1 42`
+//! + `cli command2 42`
```

Comment:
Please use something like
\```shell
cli command1 42
cli command2 42
\```

And also, could you please try to come up with something more fancy than `command1,2`? They are too similar :(

Reply:
So, what about `command1,2` stuff? It's really sad that I'm the only one who comes up with interesting names 😭